### ISSUE TITLE
CommentedOutCode should report the same comment only once

### ISSUE THREAD

**Issue by cverbiest** [Riverside-Software/sonar-openedge#556]
Sometimes, e.g when there are newlines in the commented out code, the same comment is reported multiple times by this rule

![image](https://user-images.githubusercontent.com/1312691/44142500-9d96d428-a080-11e8-8d3b-a99a2bec97e8.png)

**Sample code for multiple issues**
```
define variable lTest as character no-undo.
define variable lCount as integer no-undo.

lTest = string(lCount * 5).
if lTest > "" then lCount = 0.

/*
do lCount = 1 to 1000:

    lTest = substitute ("&1 &2", lTest, lCount).

    lTest = substitute ("&1 &2", lTest, lCount).
end.
*/
```

I also found that if a source contain only commented out code it is not reported

**Sample code for unreported issue**
```
/*
do lCount = 1 to 1000:

    lTest = substitute ("&1 &2", lTest, lCount).

    lTest = substitute ("&1 &2", lTest, lCount).
end.
*/
```

**Comment by gquerret** [comment]
Reproduced both issues.

**Comment by gquerret** [comment]
Merged in develop, will be included in next release
